src/main/java/io/github/moulberry/notenoughupdates/recipes/Ingredient.java

```diff
@@ -68,7 +68,7 @@ public static Set<Ingredient> mergeIngredients(Iterable<Ingredient> ingredients)
 			newIngredients.merge(
 				i.getInternalItemId(),
 				i,
-				(a, b) -> new Ingredient(i.manager, i.internalItemId, a.count + b.count)
+				(a, b) -> new Ingredient(i.manager, i.internalItemId, (int) (a.count + b.count))
```

Comment:
does this need the cast (shouldnt we instead change  `Ingredient(NEUManager, String, int)` to accept `double`s as well)